spark/src/main/java/org/apache/iceberg/spark/SparkUtil.java

```diff
@@ -85,17 +85,17 @@ public static void validatePartitionTransforms(PartitionSpec spec) {
 
     if (nameParts.size() == 1) {
       // Only a single element, use current catalog and namespace
-      return Pair.of(currentCatalog, identiferProvider.apply(currentNamespace, name));
+      return Pair.of(currentCatalog, identifierProvider.apply(currentNamespace, name));
     } else {
       C catalog = catalogProvider.apply(nameParts.get(0));
       if (catalog == null) {
         // The first element was not a valid catalog, treat it like part of the namespace
         String[] namespace =  nameParts.subList(0, lastElementIndex).toArray(new String[0]);
-        return Pair.of(currentCatalog, identiferProvider.apply(namespace, name));
+        return Pair.of(currentCatalog, identifierProvider.apply(namespace, name));
       } else {
         // Assume the first element is a valid catalog
         String[] namespace = nameParts.subList(1, lastElementIndex).toArray(new String[0]);
-        return Pair.of(catalog, identiferProvider.apply(namespace, name));
+        return Pair.of(catalog, identifierProvider.apply(namespace, name));
```

Comment:
This looks like an unrelated change. Can you move it to a separate PR? We want to keep this one small since it's quite large already.

Reply:
I was calling into this code and took the opportunity to fix a typo. I'll move this typo fix into a separate PR.